styles/widgets/common/fileManager.less

```diff
@@ -135,12 +135,19 @@
         overflow: hidden;
 
         .dx-filemanager-dirs-panel {
-            min-width: 250px;
-            max-width: 300px;
             padding: 5px 10px;
+            height: 100%;
+            min-width: 250px;
+            max-width: unset;
```

Comment:
How good it looks on wide screens?

Reply:
By default dirs panel is rendered with 250px width, as far as I could explore on my only monitor.